core/config/settings.yml

```diff
@@ -1,3 +1,4 @@
 core:
     debug: false
     log_file: ~
+    mode: production
```

Comment:
in addition, I think we should introduce a specific flag for serving minified content, so `minified_assets` or something similar